VARIABLES.md

```diff
@@ -14,6 +14,8 @@
 | apiary_rds_additional_sg | Comma-separated string containing additional security groups to attach to RDS. | list | `<list>` | no |
 | apiary_shared_schemas | Schema names which are accessible from read-only metastore, default is all schemas. | list | `<list>` | no |
 | apiary_tags | Common tags that get put on all resources. | map | - | yes |
+| atlas_kafka_bootstrap_servers | Atlas kafka bootstrap servers. | string | `` | no |
+| atlas_cluster_name | Name of the Atlas cluster where metastore plugin will send DDL events.  Defaults to `var.instance_name` if not set. | string | `` | no |
```

Comment:
All of the changes in this file _except_ for the above line was just to re-alphabetize the variables.